api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -190,6 +190,17 @@ message JwtProvider {
   //       exp: 1501281058
   //
   string payload_in_metadata = 9;
+
+  // The two below fields define the amount of slack in seconds that will be used
+  // when determining if a JWT is valid or has expired. Validity is determined by
+  // the formula: VALID_FROM("iat") + nbf_slack <= NOW <= VALID_TO("exp") + exp_slack
```

Comment:
s/iat/nbf

Reply:
Still seeing this:

source/extensions/filters/http/jwt_authn/authenticator.cc:172:59: error: no member named 'nbf_slack' in 'envoy::extensions::filters::http::jwt_authn::v3::JwtProvider'
  const uint32_t nbf_slack = jwks_data_->getJwtProvider().nbf_slack();
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
source/extensions/filters/http/jwt_authn/authenticator.cc:173:59: error: no member named 'exp_slack' in 'envoy::extensions::filters::http::jwt_authn::v3::JwtProvider'
  const uint32_t exp_slack = jwks_data_->getJwtProvider().exp_slack();

Reply:
Hmm,  make sure this modified proto files are compiled. by

bazel build //api/...

for docker run: you need to run this command "bazel.api"

Reply:
I am seeing the error as output from
https://circleci.com/gh/envoyproxy/envoy/338165?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification